files/blocks/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "blocks",
+  "repo": "astoilkov/jsblocks",
+  "files": {
+    "basePath": "dist/"
+  }
```

Comment:
Well, jsDelivr is for production websites. Source code is not relevant there, the most important is speed. My original config should work fine for that if you agree